Client/Frontend/Fakespot/FakespotOptInCardViewModel.swift

```diff
@@ -0,0 +1,121 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+import Common
+import Shared
+import ComponentLibrary
+
+struct FakespotOptInCardViewModel {
+    private struct UX {
+        static let contentStackViewPadding: CGFloat = 16
+        static let bodyFirstParagraphLabelFontSize: CGFloat = 15
+    }
+    private let tabManager: TabManager
```

Comment:
nit; spacing just so the code breaths a little bit more
```suggestion
    }

    private let tabManager: TabManager
```